src/Confluent.Kafka/Consumer.cs

```diff
@@ -89,18 +89,8 @@ public class Consumer<TKey, TValue> : IDisposable
 
             consumer = new Consumer(config);
 
-            // TODO: bypass this.consumer for this event to optimize perf.
```

Comment:
removed this because it is never called because the `Poll` method on the non-deserializing consumer is never called. It's hanging around from earlier times.